playground/components/editor.tsx

```diff
@@ -15,6 +17,38 @@ import { GridResizer } from './grid-resizer';
 import { FrameworkSwitcher } from './framework-switcher';
 import type { Framework } from '@/types';
 
+const zFrameworks = z.enum(['react', 'nextjs']);
+
```

Comment:
I think we could even move this to `@/atoms/framework` and infer the `Framework` type from the zod schema, this way we don't need to update multiple places every time we add a new preset/framework, any thoughts?

Reply:
totally agree will change